d/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM dlang2/ldc-ubuntu AS build
```

Comment:
why this change ? what is the common usage in dlang community ?

Reply:
It seems this updated docker image needs some extra packages to be installed to run applications. I am revert to the previous one and run CI tests in my local branch. It works for everything except handy-http - probably because the compiler version too old and doesn't support some changes that were made in newer releases. Let me try to fix it somehow.